release-notes-3.1.1.md

```diff
@@ -0,0 +1,55 @@
+# Release Notes
+
+While the 3.1.1 release makes no mechanical changes to the OpenAPI 3.1.0 specification, it does introduce a number of notable improvements, including:
+- Expands and clarifies a number of explanations, including several new appendices with supplementary details
+- Focuses on technical specifics by moving examples and additional documentation now published at https://learn.openapis.org
+- Declares that the HTML specifications at https://spec.openapis.org are now the authoritative versions (formerly it was the Markdown source on GitHub)
+
+OpenAPI description writers should mark their OpenAPI descriptions with the version of the OpenAPI specification they used to write their specification, updating where possible.
```

Comment:
Mike's PR standardized on capitalizing "Description" so perhaps we should also do that here.